src/style.css

```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+input#big {
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 19px 19px;
+  background-position: 25px 24px;
+}
+
+input#small {
+  background: url(images/Search.svg) no-repeat;
+  background-position: 12px 14px;
+  background-size: 11px 11px;
+}
```

Comment:
Use classes instead of tag names for styling

Also according to [checklist ](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)you have to create additional class-midifiers for small form
![image](https://user-images.githubusercontent.com/80700247/227746898-d22c6662-2ae1-47d1-aeec-08c10dbe8b86.png)